test/http-error.ts

```diff
@@ -1,18 +1,18 @@
 import test from 'ava';
 import {HTTPError} from '../source/index.js';
 
+type Mutable<T> = {
```

Comment:
Let me know if I can move it to some place to reuse it (was used twice in the codebase)

Reply:
You can move the type utilities to a `types` file in https://github.com/sindresorhus/ky/tree/main/source/utils

Reply:
Moved to source/utils ✅